src/test/java/nextstep/subway/acceptance/LineSectionAcceptanceTest.java

```diff
@@ -28,67 +38,131 @@
     public void setUp() {
         super.setUp();
 
-        강남역 = 지하철역_생성_요청("강남역").jsonPath().getLong("id");
-        양재역 = 지하철역_생성_요청("양재역").jsonPath().getLong("id");
+        일번역1 = 지하철역_생성_요청한다(StationRequest.of("일번역1")).as(StationResponse.class).getId();
+        이번역2 = 지하철역_생성_요청한다(StationRequest.of("이번역2")).as(StationResponse.class).getId();
+        삼번역3 = 지하철역_생성_요청한다(StationRequest.of("삼번역3")).as(StationResponse.class).getId();
+        사번역4 = 지하철역_생성_요청한다(StationRequest.of("사번역4")).as(StationResponse.class).getId();
+
+        신분당선 = 지하철_노선_생성_요청(LineRequest.of("신분당선", "bg-red-600",
+            일번역1, 이번역2, DISTANCE_BASIC)).jsonPath().getLong("id");
+        지하철_노선에_지하철_구간_생성_요청(신분당선, SectionRequest.of(이번역2, 삼번역3, DISTANCE_BASIC));
+        지하철_노선에_지하철_구간_생성_요청(신분당선, SectionRequest.of(삼번역3, 사번역4, DISTANCE_BASIC));
 
-        Map<String, String> lineCreateParams = createLineCreateParams(강남역, 양재역);
-        신분당선 = 지하철_노선_생성_요청(lineCreateParams).jsonPath().getLong("id");
+        신규역 = 지하철역_생성_요청한다(StationRequest.of("신규역")).as(StationResponse.class).getId();
     }
 
-    /**
-     * When 지하철 노선에 새로운 구간 추가를 요청 하면
-     * Then 노선에 새로운 구간이 추가된다
-     */
-    @DisplayName("지하철 노선에 구간을 등록")
+    @DisplayName("기존 지하철 노선 뒤에 구간 추가 성공하는 단순 케이스 (해피케이스)")
     @Test
-    void addLineSection() {
+    void addLineSection_성공케이스_해피케이스() {
         // when
-        Long 정자역 = 지하철역_생성_요청("정자역").jsonPath().getLong("id");
-        지하철_노선에_지하철_구간_생성_요청(신분당선, createSectionCreateParams(양재역, 정자역));
+        지하철_노선에_지하철_구간_생성_요청(신분당선, SectionRequest.of(사번역4, 신규역, DISTANCE_VALID));
 
         // then
         ExtractableResponse<Response> response = 지하철_노선_조회_요청(신분당선);
         assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
-        assertThat(response.jsonPath().getList("stations.id", Long.class)).containsExactly(강남역, 양재역, 정자역);
+        assertThat(response.jsonPath().getList("stations.id", Long.class))
+            .containsExactly(일번역1, 이번역2, 삼번역3, 사번역4, 신규역);
     }
 
-    /**
-     * Given 지하철 노선에 새로운 구간 추가를 요청 하고
-     * When 지하철 노선의 마지막 구간 제거를 요청 하면
-     * Then 노선에 구간이 제거된다
-     */
-    @DisplayName("지하철 노선에 구간을 제거")
+    @DisplayName("지하철 노선에 상행선쪽에 구간 등록을 성공")
     @Test
-    void removeLineSection() {
-        // given
-        Long 정자역 = 지하철역_생성_요청("정자역").jsonPath().getLong("id");
-        지하철_노선에_지하철_구간_생성_요청(신분당선, createSectionCreateParams(양재역, 정자역));
+    void addLineSection_성공케이스_1() {
+        // when
+        지하철_노선에_지하철_구간_생성_요청(신분당선, SectionRequest.of(신규역, 일번역1, DISTANCE_VALID));
+
+        // then
+        ExtractableResponse<Response> response = 지하철_노선_조회_요청(신분당선);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(response.jsonPath().getList("stations.id", Long.class))
+            .containsExactly(신규역, 일번역1, 이번역2, 삼번역3, 사번역4);
```

Comment:
```suggestion
        응답_요청_확인(HttpStatus.OK)
        노선_구간_정상_등록_확인(Arrays.asList(신규역, 일번역1, 이번역2, 삼번역3, 사번역4));
```
이런식으로 분리가 될 수 있을 것 같아요 👍

Reply:
옙 예시 감사합니다.
테스트를 좀 보기쉽게 리팩토링 해봐야겠습니다!